README.md

```diff
@@ -68,65 +70,35 @@ Simple Mbed Cloud Client provides Greentea tests to test your Simple Mbed Cloud
- 
+
  2. Set an environment variable on the host machine called `MBED_CLOUD_API_KEY` which is valid for the account that your device will connect to. For instructions on how to generate an API key, please [see the documentation](https://cloud.mbed.com/docs/current/integrate-web-app/api-keys.html#generating-an-api-key).
- 
- 3. In your reference application, change the following parameters in `mbed_app.json` to the parameters specific to your platform:
- ```json      
-"test-connect-header-file": {
-    "help": "Name of socket interface for SMCC tests.",
-    "value": "\"EthernetInterface.h\""
-},
-"test-socket-object": {
-    "help": "Instantiation of network interface statement for SMCC tests. (variable name must be net)",
-    "value": "EthernetInterface net;"
-},
-"test-socket-connect": {
-    "help": "Network socket connect statement for SMCC tests.",
-    "value": "net.connect();"
-},
-"test-block-device-header-file": {
-    "help": "Name of block device for SMCC tests.",
-    "value": "\"SDBlockDevice.h\""
-},
-"test-block-device-object": {
-    "help": "Block device instantiation for SMCC tests. (variable name must be bd)",
-    "value": "SDBlockDevice bd(MBED_CONF_APP_SPI_MOSI, MBED_CONF_APP_SPI_MISO, MBED_CONF_APP_SPI_CLK, MBED_CONF_APP_SPI_CS);"
-}
-```
-For example, to run the Simple Mbed Cloud Client tests on a `UBLOX_EVK_ODIN_W2`, you would add the following configuration in `target_overrides`:
+
+ 3. If your platform does not have a default network interface in `targets.json`, you can define this in `mbed_app.json`, for example:
  ```json
-"target_overrides": {
-    "UBLOX_EVK_ODIN_W2": {
-        "app.sotp-section-1-address"    : "(0x081C0000)",
-        "app.sotp-section-1-size"       : "(128*1024)",
-        "app.sotp-section-2-address"    : "(0x081E0000)",
-        "app.sotp-section-2-size"       : "(128*1024)",
-        "test-connect-header-file"      : "\"OdinWiFiInterface.h\"",
-        "test-block-device-header-file" : "\"SDBlockDevice.h\"",
-        "test-socket-object"            : "OdinWiFiInterface net;",
-        "test-socket-connect"           : "net.connect(MBED_CONF_APP_WIFI_SSID, MBED_CONF_APP_WIFI_PASSWORD, NSAPI_SECURITY_WPA_WPA2);",
-        "test-block-device-object"      : "SDBlockDevice bd(D11, D12, D13, D9);"
-    }
-}
+"target.network-default-interface-type": "ETHERNET",
 ```
-For more examples and a template `mbed_app.json`, see [simple-mbed-cloud-client-template-restricted](https://github.com/ARMmbed/simple-mbed-cloud-client-template-restricted).
 
-4. You may need to delete your `main.cpp`.
+1. If your platform does not have a default storage component in `targets.json`, you can define this in `mbed_app.json`. Aditionally, you may want to double-check the storage pinout configuration for your platform in the `mbed_lib.json` file in `mbed-os/components/storage/blockdevice/`.
+   
+
+```
```

Comment:
I believe the backticks should be removed here, they are messing up the formatting for the rest of the doc.

Reply:
Good catch! Resolved now.